runner/src/mill/runner/MillMain.scala

```diff
@@ -491,4 +517,9 @@ object MillMain {
     for (k <- systemPropertiesToUnset) System.clearProperty(k)
     for ((k, v) <- desiredProps) System.setProperty(k, v)
   }
+
+  private val bspInstallAliases = Seq(
+    "mill.bsp.BSP/install",
+    "mill.bsp/install"
```

Comment:
Under the assumption, that we also add a dedicated cli option, I would only keep the first alias. The second one was added after any stable release.

Reply:
Should be fixed